src/modules/markets/entities/market.entity.ts

```diff
@@ -59,11 +59,11 @@ export class MarketEntity {
 
   @Column({ name: 'ESRA_ID' })
   @ApiProperty({ example: 1, description: '' })
-  ESRAClasificationId: number;
+  ESRAClassificationId: number;
```

Comment:
@avaitonis  can you check if this is correct? there was a typo here

Reply:
This typo was introduced 7 months ago and endpoint is now used by EXIP. We can fix typo but release of this nodeJS API will require small change in EXIP project.

Reply:
ok cool. let's fix it as part of this PR and we will notify Tony once the API is live